src/test_l1_dsVideoDevice.c

```diff
@@ -439,6 +441,7 @@ void test_l1_dsVideoDevice_positive_dsSetDFC (void)
  * |07|Call dsSetDFC() after termination of video devices | int=handle, dsVideoZoom_t=dsVIDEO_ZOOM_NONE | dsERR_NOT_INITIALIZED | Should report module not initialized |
```

Comment:
```suggestion
 * |07|Call dsSetDFC() after termination of video devices | int=handle, dsVideoZoom_t=dsVIDEO_ZOOM_NONE | dsERR_NOT_INITIALIZED (source device) / dsERR_OPERATION_NOT_SUPPORTED (sink device) | Should report module not initialized on source devices |
```
Hi @kanjoe24, 
Before I spend the time updating these test procedures, can I please check if this format will be suitable, or if there is a different approach you would like me to take?

Reply:
You can follow this : https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/main/src/test_l1_dsDisplay.c#L895

Reply:
Hi @kanjoe24, 
I have updated the test procedures for the APIs that are not supported by sink device:
- dsSetDFC
- dsGetDFC
- dsGetVideoCodecInfo 

Please can you review these changes? If they are satisfactory I will continue to do the same with the rest of the APIs. 

Many thanks

Reply:
I have just this comment: https://github.com/rdkcentral/rdk-halif-test-device_settings/pull/440#discussion_r2135898402.

Rest looks fine to me.

Reply:
Thank you @kanjoe24 - I will do the same for the rest of the affected APIs